src/QirRuntime/test/QIR-dynamic/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_executable(qir-dynamic-tests
 target_link_libraries(qir-dynamic-tests PUBLIC
   ${QIR_TESTS_LIBS}
   qdk
+  qir-qis-support

```

Comment:
The idea of the "dynamic" tests was to validate that can link against the dynamic lib (qdk) without any knowledge about the static libs that go into it. If it's failing to link after your changes, it means that we aren't building the qdk component correctly. Ideally, should fix the build. Or log a bug and disable the component for now.

Reply:
as discussed offline: add the export decls on the new APIs

Reply:
Fixed.